README.md

```diff
@@ -64,7 +64,7 @@ This tutorial is also available in [other languages](https://github.com/Syknapse
 First let's get setup to do the work
 
 1. Login to your GitHub account. If you don't yet have an account then [join GitHub](https://github.com/join). I recommend that you do the [GitHub Hello World tutorial](https://guides.github.com/activities/hello-world/) before you continue.
-1. Download [GitHub Desktop app](https://desktop.github.com/).
+2. Download [GitHub Desktop app](https://desktop.github.com/).
```

Comment:
in markdown ordered lists we should always **all** the list items numbered as 1. Markdown automatically assigns the right number to each depending on order. This has the big advantage that if you add a new item in the middle of a long list you don't have to manually go changing the numbering on all the other items.

Reply:
done